tensorflow/python/debug/cli/analyzer_cli_test.py

```diff
@@ -46,6 +46,9 @@
 from tensorflow.python.platform import test
 from tensorflow.python.util import tf_inspect
 
+def matmul_prefix():
```

Comment:
I think making the function return the whole op name is a little more efficient (so we don't have to do the extra concat with "MatMul" every time). Would you mind changing this to `matmul_op_name` instead? Since this is in general TF code, please also add a comment in the code describing why we need this function (op name changed when Intel MKL-DNN is enabled).

Reply:
I have fixed it.